### PR TITLE
Upgrade Prusa Link to new /v1/ API

### PR DESCRIPTION
**Pull request by Skaronator**
## Proposed change

Contains changes that are relevant to my Home Assistant Core PR.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/103396
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by Skaronator** [comment]
Apologies for the confusion; I made an error in the cherry-pick process, which caused some of you to be assigned for review.

File: source/_integrations/prusalink.markdown [hunk 1]
```diff
@@ -17,16 +18,21 @@ ha_platforms:
 ha_dhcp: true
 ---
 
-The PrusaLink integration allows you to monitor your [Prusa 3D printer](https://www.prusa3d.com) and its progress with your Home Assistant installation. This integration works with Prusa MINI and Prusa MK4.
-This integration requires PrusaLink v2. If you have the Prusa MINI, you need to run firmware 4.4.0 or later.
+The PrusaLink integration allows you to monitor your [Prusa 3D printer](https://www.prusa3d.com) and its progress with your Home Assistant installation. This integration works with Prusa MINI, Prusa MK3.9/MK4 and Prusa XL. It does not work with the older Raspberry Pi Prusa Link printers.
 
-When adding the integration, you will be asked to provide a {% term host %} and an API key. These are the credentials you set up when setting up PrusaLink on your printer.
+This integration integration has been updated to utilize the latest v1 API endpoints, which require firmware version 4.7.0 or later. If you own a Prusa Mini, please make sure your printer is running firmware 5.1.0 or a more recent version, as firmware versions 4.7.x and 5.0.x are not available for this model.
```
Comment (c0ffeeca7) [inline source/_integrations/prusalink.markdown]:
```suggestion
This integration has been updated to utilize the latest v1 API endpoints, which require firmware version 4.7.0 or later. If you own a Prusa Mini, please make sure your printer is running firmware 5.1.0 or a more recent version. Firmware versions 4.7.x and 5.0.x are not available for this model.
```

File: source/_integrations/prusalink.markdown [hunk 2]
```diff
@@ -17,16 +18,21 @@ ha_platforms:
...
 
-To obtain the host name and API key/password:
+Firmware update guides can be found here:
+
+ - [Prusa Mini](https://help.prusa3d.com/article/firmware-updating-mini-mini_124784)
+ - [Prusa MK4/XL](https://help.prusa3d.com/article/how-to-update-firmware-mk4-xl_453086)
+
+To obtain the host name, username and password:
```
Comment (c0ffeeca7) [inline source/_integrations/prusalink.markdown]:
```suggestion
To obtain the hostname, username, and password:
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @Skaronator :+1:

**Comment by Skaronator** [comment]
Thanks!

**Pull request merged**